traja/tests/test_trajectory.py

```diff
@@ -221,13 +221,6 @@ def test_transition_matrix():
     transitions_matrix = traja.transition_matrix(grid_indices1D)
 
 
-def test_calc_laterality():
```

Comment:
It appears this test was unintentionally removed. Please restore it.

Reply:
Just re-added this test and pushed to this branch.  Thanks!

Reply:
Thanks!